docs/azure_jumpstart_ag/manufacturing/contoso_motors/welding_defect/_index.md

```diff
@@ -77,6 +77,26 @@ Expected color order is BGR.
 
 The features is a blob with the shape 1, 3 containing probability scores for three output classes (**no weld**, **normal weld** and **porosity**).
```

Comment:
something about this sentence seems off.

Reply:
@fcabrera23 can you help with explaining what this sentence should say ?